lambda/query-user-services/query-user-services.ts

```diff
@@ -90,7 +91,13 @@ export const handler = async (event: DynamoDBStreamEvent): Promise<void> => {
   await Promise.all(
     Records.map(async (record) => {
       try {
-        const eventBody = record.dynamodb?.NewImage?.event.S as string;
+        const eventBody = JSON.stringify(
```

Comment:
There may be a good reason for this because DynamoDB is _weird_, but calling `JSON.stringify()` on an object and then `JSON.parse()` the line after doesn't look quite right. 

Could we do it like this instead? (I've checked the unit tests pass but not deployed it)
```typescript
      try {
        const txmaEvent = unmarshall(
          record.dynamodb?.NewImage?.event.M as {
            [key: string]: AttributeValue;
          }
        ) as TxmaEvent;
        validateTxmaEventBody(txmaEvent);
```

Reply:
hmm yeah .. honestly I didn't think about this as I was going back and forth with different changes to get this working. I will deploy once and see if the lambdas are happy